src/electionguard/election_polynomial.py

```diff
@@ -107,21 +109,32 @@ def compute_lagrange_coefficient(coordinate: int, *degrees: int) -> ElementModQ:
 def verify_polynomial_coordinate(
     coordinate: ElementModQ,
     exponent_modifier: int,
-    coefficient_commitments: List[ElementModP],
+    commitments: List[PUBLIC_COMMITMENT],
 ) -> bool:
     """
     Verify a polynomial coordinate value is in fact on the polynomial's curve
 
     :param coordinate: Value to be checked
     :param exponent_modifier: Unique modifier (usually sequence order) for exponent
-    :param coefficient_commitments: Commitments for coefficients of polynomial
+    :param commitments: Public commitments for coefficients of polynomial
     :return: True if verified on polynomial
     """
+
+    _check_exponent_modifier(exponent_modifier)
+
     commitment_output = ONE_MOD_P
-    for (i, commitment) in enumerate(coefficient_commitments):
+    for (i, commitment) in enumerate(commitments):
         exponent = pow_p(int_to_p_unchecked(exponent_modifier), int_to_p_unchecked(i))
         factor = pow_p(commitment, exponent)
         commitment_output = mult_p(commitment_output, factor)
 
     value_output = g_pow_p(coordinate)
     return value_output == commitment_output
+
+
+def _check_exponent_modifier(exponent_modifier: int) -> None:
```

Comment:
why not just call `ElementModQ.is_in_bounds()` instead of making this method?

Reply:
This can be done for the high bound but the actual importance of this method is the lower end. There is an active PR that also addresses this. This is in relation to the fact that sequence order cannot be zero.